plugins/dummy_plugin.py

```diff
@@ -35,7 +35,6 @@ def request_http(self, query, user_id, test_end_time: float=0):
 
         result.end_time = time.time()
 
-        result.calculate_results()
```

Comment:
When we do a cleanup we probably should remove this file.

Reply:
Yeah, this was originally added with the thought that it could be used in some test cases but we may want to remove it depending on how we decide to handle testing (unit tests, e2e tests, etc...)